pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -1445,7 +1445,7 @@ self: {
        license = stdenv.lib.licenses.bsd3;
        hydraPlatforms = stdenv.lib.platforms.none;
        broken = true;
-     }) {inherit (pkgs) db; inherit (pkgs) dbxml; 
```

Comment:
This file is generated automatically, so all these changes will get reverted next time it is automatically generated.

Reply:
What do we need to do that the generator uses `mpi` instead of `openmpi`?